app/views/users/mod_delete.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :title, "Moderator Tools: #{rtl_safe_username(@user)}" %>
+
+<%= render 'tabs', user: @user %>
+
+<div class="grid">
+  <%= render 'shared/user_mod_sidebar', user: @user %>
+
+  <div class="grid--cell is-9">
+    <h1>Delete or Destroy Account</h1>
+
+    <p>Some users are just blatant spammers or trolls and some users are just unwilling to follow site rules, even after repeated warnings and suspensions. As a moderator, you may delete the user account in these cases.</p>
+
+    <div class="notice is-danger">
+        <p><strong>Take care!</strong> These actions may not be reversible and you will not be asked to confirm after initiating an action.</p>
```

Comment:
Even if the option to destroy is removed, this warning will still apply to mod deleting a user from a specific community. Rather than saying "you will not be asked to confirm", would it be better to ask to confirm?